src/client/network.rs

```diff
@@ -76,25 +59,15 @@ pub enum NodeSystemCmd {
         /// Current holders.
         current_holders: BTreeSet<XorName>,
     },
-    /// When new section key, all propose new wallet.
-    ProposeNewWallet {
-        /// The genesis credit.
-        credit: Credit,
-        /// An individual Elder's sig share.
-        sig: SignatureShare,
-    },
-    /// When proposal has been agreed, they all accumulate the wallet.
-    AccumulateNewWallet {
-        /// The genesis credit.
-        signed_credit: SignedCredit,
-        /// An individual Elder's sig share.
-        sig: SignatureShare,
-    },
+    /// When new section key, all propose a reward payout.
+    ProposeRewardPayout(sn_data_types::RewardProposal),
+    /// When proposal has been agreed, they all accumulate the reward payout.
+    AccumulateRewardPayout(sn_data_types::RewardAccumulation),
     /// Sent to all promoted nodes (also sibling if any) after
     /// a completed transition to a new constellation.
     ReceiveExistingData {
         /// Registered node reward wallets.
```

Comment:
can you note a bit more what the u8 is about here?

Reply:
Ah, naturally, hang on